rllib/env/wrappers/tests/test_recsim_wrapper.py

```diff
@@ -1,14 +1,15 @@
 import gym
 import unittest
 
-from ray.rllib.env.wrappers.recsim_wrapper import (
-    MultiDiscreteToDiscreteActionWrapper, make_recsim_env)
+from ray.rllib.examples.env.recsim_recommender_system_envs import \
+    InterestEvolutionRecSimEnv
+from ray.rllib.env.wrappers.recsim import MultiDiscreteToDiscreteActionWrapper
 from ray.rllib.utils.error import UnsupportedSpaceException
 
 
 class TestRecSimWrapper(unittest.TestCase):
     def test_observation_space(self):
-        env = make_recsim_env(config={})
+        env = InterestEvolutionRecSimEnv()
         obs = env.reset()
```

Comment:
we could also check if an observation that has been sampled from the observation space is contained, although im not sure how good a test that is. We could sampled a few thousand observations and check if they're contained in the obs space (its a relatively cheap operation)

Reply:
I think we are doing this right below. Only twice, though after the `env.reset()` and the `step()`.

Added some more checks for the action space test.

Not sure I fully understand what you are aiming at, though. But please feel free to suggest more checks here.